### PR TITLE
Test that 'blocking' keyword is respected as a link header

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

**Comment by domenic** [comment]
@xiaochengh to review this. Apparently Chrome does not pass this yet, so maybe the spec should not require it after all? Or maybe that's just a Chrome bug. Please let us know.

**Comment by noamr** [comment]
> @xiaochengh to review this. Apparently Chrome does not pass this yet, so maybe the spec should not require it after all? Or maybe that's just a Chrome bug. Please let us know.

Currently no browser supports this. I added a test more as a future reference thing. Maybe it should be in the spec once there's consensus to implement this?

**Comment by domenic** [comment]
There's general consensus for `blocking=""` on `<link>`, and an implementation in Chromium. The question is whether that consensus carries over to `Link`.

**Comment by noamr** [comment]
> There's general consensus for `blocking=""` on `<link>`, and an implementation in Chromium. The question is whether that consensus carries over to `Link:`.

Yea, that's what I meant.

**Review by xiaochengh — APPROVED** [review]
Code change LGTM. Should we revise the spec to add `blocking` to link headers?

**Comment by domenic** [comment]
Link headers were not specced, but in https://github.com/whatwg/html/pull/7622 @noamr is speccing them and the idea is to make them respect all the same attributes :). So after that gets merged (momentarily), the spec will say that Link headers + blocking should work.

File: html/dom/render-blocking/header-inserted-preload-link.tentative.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@font-face {
+  font-family: custom-font;
+  src: url('/fonts/Ahem.ttf?pipe=trickle(d1)');
+}
+</style>
+<span id="target" style="font: 20px/1 custom-font">Lorem ipsum</span>
+
+<script>
+test_render_blocking(
+    null,
```
Comment (xiaochengh) [inline html/dom/render-blocking/header-inserted-preload-link.tentative.html:17]: I missed it in the code review...

`test_render_blocking` needs to be able to tell if the render-blocking resource has been loaded. It does so by listening to the `load` event of the element, or if that's impossible (e.g., a script-blocking stylesheet or a parser-blocking script), then it listens to `window.load`.

But here we don't have an element to listen to, and I guess preload link headers do not block `window.load`? Then the test won't work.

Comment (domenic) [reply html/dom/render-blocking/header-inserted-preload-link.tentative.html:17]: Hmm. Do you have a suggested better way of testing this? Also curious to get @noamr's thoughts.

**Pull request merged**